server/docker/config/app-config.toml

```diff
@@ -6,5 +6,5 @@ log_level = "info"
 [mail_config]
 smtp_address = "mailhog"
```

Comment:
`smtp_address` should not be 'mailhog' if the container name changed. This won't work like this, I think.